frontend/app/api/index.js

```diff
@@ -208,5 +208,12 @@ module.exports = function (app, port) {
         });
       }, 500);
     }
-  )
+  );
+
+  /*
+    VERSION
+  */
+  app.get('/api/version', function(req, res) {
+    res.sendFile(path.join(__dirname, './version.json'))
```

Comment:
Our demo backend is fake in a sense - but we might not have to fake this response. What about `require`ing the `package.json` and reading the npm package version from there? The `isDevelopment` value can also be easily obtained in node...

Reply:
@manuel-hegner wanted to send the value from the backend. In a production build we do not have the package.json but the version is still displayed as title tag of the conquery logo.

Reply:
This is demo backend code -- we sure have the package.json when running that backend in production!

Reply:
Sorry, misunderstood you. You were completely right and I changed it using the package.json and NODE_ENV